api/app/wildfire_one/schema_parsers.py

```diff
@@ -273,7 +275,9 @@ def generate_station_daily(raw_daily,  # pylint: disable=too-many-locals
         sixty_minute_fire_size=sixty_minute_fire_size,
         fire_type=fire_type,
         error=raw_daily.get('observationValidInd', None),
-        error_message=raw_daily.get('observationValidComment', None)
+        error_message=raw_daily.get('observationValidComment', None),
+        last_updated=datetime.fromisoformat(raw_date_string[:len(
```

Comment:
What happens if we just set this as `raw_daily.get('updateDate', None)` ?

Reply:
It returns a date string that isn't iso compatible.

Reply:
Yuck. Ok, WFWX also returns an update timestamp in milliseconds, it's called `lastEntityUpdateTimestamp`.
https://chat.developer.gov.bc.ca/channel/wildfire-wfwx?msg=BvE29dpw6JuXTmd3i

So we can just do:
```
updateTimestamp = int(raw_daily.get('lastEntityUpdateTimestamp'))
last_updated=datetime.fromtimestamp(updateTimestamp / 1000, tz=timezone.utc)
```